File: src/main/java/src/SistemaReservaHotel/Habitacion.java
package SistemaReservaHotel;

public class Habitacion {
    private int id;
    private boolean disponible;

    public Habitacion(int id) {
        this.id = id;
        this.disponible = true;
    }

    public int getId() {
        return id;
    }

    public boolean isDisponible() {
        return disponible;
    }

    public void setDisponible(boolean disponible) {
        this.disponible = disponible;
    }

    @Override
    public String toString() {
        return "Habitación{" +
                "ID=" + id +
                ", Disponible=" + disponible +
                '}';
    }
}

File: src/main/kotlin/SistemaReservaHotel/Habitacion.kt
package SistemaReservaHotel

class Habitacion(val id: Int) {
    var disponible: Boolean = true

    fun changeDisponible(disponible: Boolean) {
        this.disponible = disponible
    }

    override fun toString(): String {
        return "Habitación{" +
                "ID=" + id +
                ", Disponible=" + disponible +
                '}'
    }
}